.github/workflows/codecov.yaml

```diff
@@ -1,6 +1,8 @@
 name: Code Coverage with codecov
 
 on:
+  workflow_dispatch:
+    branches: [main]
```

Comment:
It's still here in the first commit. I guess you squashed the removal into the wrong commit?

Reply:
I did not squash, thats why I mentioned it in the commit message where it was removed.

Reply:
anyways, squashed now.